server/internal/capture/streamsink.go

```diff
@@ -169,7 +182,7 @@ func (manager *StreamSinkManagerCtx) createPipeline() error {
 				return
 			}
 
-			manager.sampleFn(sample)
+			manager.sampleChannel <- sample
```

Comment:
In that case this whole goroutine is not needed. It only converted from channel to event. We could pass the cannel straight down to the receiver.

Reply:
changed.